network-api/networkapi/wagtailpages/utils.py

```diff
@@ -31,7 +31,9 @@ def get_descendants(node, list, depth=0, max_depth=2):
     discovery pass of all menu-listable children to some root node.
     '''
     if (depth <= max_depth):
-        title = node.specific.header if node.specific.header else node.title
+        title = node.title
```

Comment:
this change was necessary due to `RedirectingPage` not having a header attribute, so it would literally crash Django when trying to build the menu.